src/components/ProductCard/ProductCard.jsx

```diff
@@ -0,0 +1,37 @@
+import "./productcard.css";
+
+const ProductCard = ({product}) => {
+
+    console.log(product);
+
+    const {title, description, image, price, discountedPrice } = product;
+    // rating,
+        // inStock,
+        // fastDelivery,
+
```

Comment:
remove it if it's not intentional.

Reply:
Thanks for pointing out, Gopal. 
I'll do that.